bees/devrantbee/devrantbee.go

```diff
@@ -81,61 +83,30 @@ func (mod *DevrantBee) Action(action bees.Action) []bees.Placeholder {
 }
 
 func (mod *DevrantBee) triggerEvent(rant devgorant.RantModel) {
+	v := reflect.ValueOf(rant)
+	names := make([]string, v.NumField())
+	types := make([]string, v.NumField())
+	values := make([]interface{}, v.NumField())
+
+	opts := make([]bees.Placeholder, v.NumField())
+
+	for i := 0; i < v.NumField(); i++ {
+		names[i] = v.Type().Field(i).Name
+		types[i] = v.Field(i).Type().String()
+		values[i] = v.Field(i).Interface()
+
+		// Parsing the values into a bees.Placeholder struct
+		opts[i] = bees.Placeholder{
+			Name:  names[i],
+			Type:  types[i],
```

Comment:
I already knew this problem exists (as the title says it's just a proof of concept ;p) but I'll work on a solution to handle nested structs, arrays et cetera in a pleasant way.